code/datums/mutations/actions.dm

```diff
@@ -104,59 +104,59 @@
+	power = /obj/effect/proc_holder/spell/cone/staggered/firebreath
 	instability = 30
 	energy_coeff = 1
 	power_coeff = 1
 
 /datum/mutation/human/firebreath/modify()
-	if(power)
-		var/obj/effect/proc_holder/spell/aimed/firebreath/S = power
-		S.strength = GET_MUTATION_POWER(src)
+	// If we have a power chromosome, buff our spell
+	if(power && GET_MUTATION_POWER(src) > 1)
+		var/obj/effect/proc_holder/spell/cone/staggered/firebreath/our_spell = power
+		our_spell.cone_levels *= 2 //Power firebreath makes it fwoosh double distance.
 
-/obj/effect/proc_holder/spell/aimed/firebreath
+/obj/effect/proc_holder/spell/cone/staggered/firebreath
 	name = "Fire Breath"
-	desc = "You can breathe fire at a target."
+	desc = "You can breathe a cone of fire at a target."
 	school = SCHOOL_EVOCATION
+	invocation = ""
 	charge_max = 600
 	clothes_req = FALSE
 	range = 20
-	projectile_type = /obj/projectile/magic/aoe/fireball/firebreath
 	base_icon_state = "fireball"
 	action_icon_state = "fireball0"
+	still_recharging_msg = "<span class='warning'>You can't muster any flames!</span>"
 	sound = 'sound/magic/demon_dies.ogg' //horrifying lizard noises
-	active_msg = "You built up heat in your mouth."
-	deactive_msg = "You swallow the flame."
-	var/strength = 1
+	respect_density = TRUE
+	cone_levels = 2
 
-/obj/effect/proc_holder/spell/aimed/firebreath/before_cast(list/targets)
-	. = ..()
-	if(iscarbon(usr))
-		var/mob/living/carbon/C = usr
-		if(C.is_mouth_covered())
-			C.adjust_fire_stacks(2)
-			C.IgniteMob()
-			to_chat(C,span_warning("Something in front of your mouth caught fire!"))
-			return FALSE
-
-/obj/effect/proc_holder/spell/aimed/firebreath/ready_projectile(obj/projectile/P, atom/target, mob/user, iteration)
+/obj/effect/proc_holder/spell/cone/staggered/firebreath/before_cast(list/targets)
 	. = ..()
-	if(!istype(P, /obj/projectile/magic/aoe/fireball))
+	if(!iscarbon(usr))
+		return
+
+	var/mob/living/carbon/our_lizard = usr
+	if(!our_lizard.is_mouth_covered())
 		return
-	var/obj/projectile/magic/aoe/fireball/F = P
-	switch(strength)
-		if(1 to 3)
-			F.exp_light = strength-1
-		if(4 to INFINITY)
-			F.exp_heavy = strength-3
-	F.exp_fire += strength
-
-/obj/projectile/magic/aoe/fireball/firebreath
-	name = "fire breath"
-	exp_heavy = 0
-	exp_light = 0
-	exp_flash = 0
-	exp_fire= 4
+
+	our_lizard.adjust_fire_stacks(cone_levels)
+	our_lizard.IgniteMob()
+	to_chat(our_lizard, span_warning("Something in front of your mouth caught fire!"))
+
+/obj/effect/proc_holder/spell/cone/staggered/firebreath/do_turf_cone_effect(turf/target_turf, level)
+	// Further turfs experience less exposed_temperature and exposed_volume
+	new /obj/effect/hotspot(target_turf)
+	target_turf.hotspot_expose(max(500, 900 - (100 * level)), max(50, 200 - (50 * level)), 1)
+
+/obj/effect/proc_holder/spell/cone/staggered/firebreath/do_mob_cone_effect(mob/living/target_mob, level)
+	// Further out targets take less burn damage
+	target_mob.apply_damage(max(10, 30 - (5 * level)), BURN)
```

Comment:
ngl, this is a really disappointing amount of damage

Reply:
In close range, it's on par with the space dragon's fire breath, which is what the damage was based off of. It also applies firestacks + the hotspot, so it causes a good about of residual burns.

Reply:
Also consider its gone from single target to "Nukies all dead"

Reply:
> In close range, it's on par with the space dragon's fire breath, which is what the damage was based off of. It also applies firestacks + the turf hotspot, so it causes a good about of residual burns.

yeah, but space dragons aren't at risk of catching on fire from the people they light on fire with their own fire breath

Reply:
ignore ath he is a powergamer